tests/integration/backward_compatible/authentication_tests/authentication_test.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+from hazelcast.errors import HazelcastError
+from tests.base import HazelcastTestCase
+from tests.util import get_abs_path, set_attr
+from hazelcast.client import HazelcastClient
+from hazelcast.token_provider import BasicTokenProvider
+
+
+@set_attr(enterprise=True)
+class AuthenticationTest(HazelcastTestCase):
```

Comment:
We should also add this for the backward compatibility tests
```
@unittest.skipIf(
    is_client_version_older_than("4.2.1"), "Tests the features added in 4.2.1 version of the client"
)
```

Reply:
OK: https://github.com/hazelcast/hazelcast-python-client/pull/446/commits/36946e76b40c9e937d31840499db6bd23f1b14c6 and https://github.com/hazelcast/hazelcast-python-client/pull/446/commits/16b71978621c5fca41e3ed84810d563834847c91